pipeline/splicing/calcVarPriors.py

```diff
@@ -1119,27 +1554,36 @@ def getVarDict(variant, boundaries):
                "varGenCoordinate": variant["Pos"],
                "varType": varType,
                "varLoc": varLoc,
-               "varHGVScDNA": variant["pyhgvs_cDNA"]}
+               "varHGVScDNA": variant["HGVS_cDNA"]}
```

Comment:
curious about the change here

Reply:
I changed this because in the other file that in this PR, protein priors are found by searching for HGVS_cDNA in the file so I thought it would be easier to just switch entirely to HGVS_cDNA